sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetFileFormat.scala

```diff
@@ -298,13 +315,14 @@ class GpuParquetWriter(
     dataSchema: StructType,
     compressionType: CompressionType,
     dateTimeRebaseException: Boolean,
+    int96RebaseException: Boolean,
     context: TaskAttemptContext)
   extends ColumnarOutputWriter(path, context, dataSchema, "Parquet") {
 
   val outputTimestampType = conf.get(SQLConf.PARQUET_OUTPUT_TIMESTAMP_TYPE.key)
 
   override def scanTableBeforeWrite(table: Table): Unit = {
-    if (dateTimeRebaseException) {
+    if (dateTimeRebaseException || int96RebaseException) {
```

Comment:
Why does this class take two booleans for rebase exceptions but treat them as if they were a single one?  The INT96 config is specific to the case when INT96 is being used, and I don't see this checking for that specific type. It looks like if a rebase is needed on a date column then this config could trigger an exception despite the other config (that should apply to date columns) indicates it should not throw.

Reply:
`int96RebaseException` is taking the `parquetOutputTimestampType` into account. But I agree that I have to separate the exception checking for both of these booleans because if INT96 is set to `CORRECTED` it shouldn't check the timestamp columns for exceptions. Let me fix that.